doc/reference/building-and-signing-tx.md

```diff
@@ -6,14 +6,16 @@ Transactions vary in complexity, depending on their intended outcomes, but all t
 * Output - determine where the funds go to. An output is given by a payment address and an amount. A transaction can have multiple outputs.
 * Payment address - an address that can receive payments, This is the only type of addresses that can be specified in a transaction output.
 * Payment and stake key pairs - sets of files containing a public verification key and a private signing key.
-* Time-to-live (TTL) - represents a slot, or deadline by which a transaction must be submitted. The TTL is an absolute slot number, rather than a relative one, which means that the --ttl value should be greater than the current slot number. A transaction becomes invalid once its ttl expires.
+* Invalid-before - The slot that the transaction is valid from. This can only be specified in the Allegra era and onwards.
+* Invalid-hereafter - represents a slot, or deadline by which a transaction must be submitted. This is an absolute slot number, rather than a relative one, which means that the --invalid-hereafter value should be greater than the current slot number. A transaction becomes invalid at the invalid-hereafter slot.
```

Comment:
Add back ticks around "--invalid-hereafter" like this for monospace formatting: `--invalid-hereafter`